.changeset/spicy-frogs-press.md

```diff
@@ -0,0 +1,5 @@
+---
+'verdaccio-htpasswd': major
+---
+
+feat: allow other password hashing algorithms (#1917)
```

Comment:
Would you mind extend your instructions here (that's the beauty of changesets customizeble changelogs), eg: https://github.com/verdaccio/verdaccio/pull/2070/files#diff-8d3dd8f006a8564479376ddec2b748fe665f6c5210d16bc8b67307f9503c87b5R9 

This will be eventually part of the migration guides and documentation, so better :upside_down_face: do it only once. 

I'd recommend ad some example, which one is default, notes you consider important to know in the future.

Reply:
Done :)